docs/Contributing.asciidoc

```diff
@@ -76,6 +76,17 @@ implications and how we can help each other to improve
 If this is too much hassle for you feel free to provide incomplete pull
 requests for consideration or create an issue with a code change proposal.
 
+=== Code style suggestions
+[id="code_style_suggestions"]
+
+* In Perl files sort the use statements in this order from top to bottom:
+** "strict", "warnings" or other modules that provide static checks
+** All external modules and from "lib" folder
+** `use FindBin; use lib "$FindBin::Bin/lib";`
```

Comment:
It isn't always exactly this line.

Reply:
ok, I added "or similar to resolve internal modules".